apparmor.d/groups/virt/k3s

```diff
@@ -0,0 +1,176 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2022 Jeroen Rijken
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/3.0>,
+
+include <tunables/global>
+
+@{exec_path} = /{usr/,}{local/,}bin/k3s
+profile k3s @{exec_path} flags=(complain) {
+  include <abstractions/base>
+  include <abstractions/ssl_certs>
+  include <abstractions/nameservice-strict>
+  include <abstractions/disks-read>
+
+  capability chown,
+  capability dac_override,
+  capability dac_read_search,
+  capability net_admin,
+  capability syslog,
+  capability sys_admin,
+  capability sys_resource,
+
+  ptrace peer=@{profile_name},
+  ptrace (read) peer=unconfined,
+
+  network inet    dgram,
+  network inet6   dgram,
+  network inet    stream,
+  network inet6   stream,
+  network netlink raw,
+
+  mount  /var/lib/kubelet/pods/@{uuid}/volumes/kubernetes.io~*/{,**/},
+  umount /var/lib/kubelet/pods/@{uuid}/volumes/kubernetes.io~*/{,**/},
+
+  signal (send, receive) set=term,
+
+  @{exec_path} mr,
+  /{usr/,}bin/kmod rPx,
+  /{usr/,}bin/mount rPx,
+  /{usr/,}bin/systemd-run rix,
+
+  # These are all symbolic links to xtables-nft-multi on Ubuntu 22.04
+  /{usr/,}{s,}bin/iptables rPx -> xtables-nft-multi,
+  /etc/alternatives/iptables rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/iptables-legacy rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/xtables-nft-multi rPx,
+
+  /{usr/,}{s,}bin/iptables-save rPx -> xtables-nft-multi,
+  /etc/alternatives/iptables-save rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/iptables-legacy-save rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/xtables-nft-multi rPx,
+
+  /{usr/,}{s,}bin/iptables-restore rPx -> xtables-nft-multi,
+  /etc/alternatives/iptables-restore rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/iptables-legacy-restore rPx -> xtables-nft-multi,
+  /{usr/,}{s,}bin/xtables-nft-multi rPx,
+
+  /usr/libexec/kubernetes/kubelet-plugins/volume/exec/nodeagent~uds/uds rix,
+  /var/lib/rancher/k3s/data/[0-9a-f]*/bin/* rix,
+
+  /usr/libexec/kubernetes/kubelet-plugins/volume/exec/{,**} r,
```

Comment:
Always use `@{libexec}` instead of `/usr/libexec` as some distributions do not have `/usr/libexec`
```suggestion
  @{libexec}/kubernetes/kubelet-plugins/volume/exec/{,**} r,
```